projects/plugins/jetpack/_inc/client/components/jetpack-product-card/index.jsx

```diff
@@ -91,6 +98,20 @@ const JetpackProductCard = props => {
 				<Button className={ buttonClasses } href={ checkoutUrl } onClick={ onClick }>
 					{ checkoutText }
 				</Button>
+
+				{ disclaimer && (
+					<p className="jp-product-card__disclaimer">
+						{ `${ disclaimer.text } ` }
+						<a
+							onClick={ onDisclaimerClick }
+							href={ disclaimer.url }
+							target="_blank"
+							rel="noreferrer"
+						>
```

Comment:
I wonder if we should be using the `<ExternalLink>` component here?  Most of all the other links throughout the plugin that point to an external target are using the `ExternalLink` component ,which adds a little icon indicating to the user that it is external and it opens in a new window.

Reply:
Was unaware of this component, thanks 😄